android/KMEA/app/src/main/java/com/tavultesoft/kmea/DisplayLanguages.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (C) 2021 SIL International. All rights reserved.
+ */
+
+package com.tavultesoft.kmea;
+
+public class DisplayLanguages {
+  private static final String TAG = "DisplayLanguages";
+  public static final String displayLanguageKey = "DisplayLanguage";
+  public static final String unspecifiedLocale = ""; // empty string for app to use device's default locale
+
+  public static class DisplayLanguageType {
+    String languageTag;
+    String languageName;
+
+    public DisplayLanguageType(String languageTag, String languageName) {
+      this.languageTag = languageTag;
+      this.languageName = languageName;
+    }
+
+    public String getLanguageTag() {
+      return this.languageTag;
+    }
+
+    public String getLanguageName() {
+      return this.languageName;
+    }
+  }
+
+  // Display Language types (as named in translate.keyman.com)
+  // Order doesn't matter since we're storing BCP-47 tags in the settings preference
+  public static final DisplayLanguageType[] DisplayLanguages =
+  {
+    new DisplayLanguageType(unspecifiedLocale, "Default Locale"),
+    new DisplayLanguageType("km-KH", "Khmer"),
+    new DisplayLanguageType("ann", "Obolo")
+  };
```

Comment:
I take it this will need to be maintained manually? We need to document this as I suspect it will be forgotten otherwise!

Reply:
Yes, this is maintained manually. I've added a note in resources/build/l10n/README.md